package.json

```diff
@@ -121,7 +124,7 @@
     "rxjs": "^6.3.3"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
@sharkparty could this be where the `0.10.x` is coming from?

Reply:
I bumped this back up and defined the version of node in the travis yaml file. Idiot me didn't think that travis would default to a prehistoric version of node. Surprise surprise. I also had deleted the docs directory to do some testing and didn't bring that back, so I added it back. All is good now.